# request 
bool take_snap
---
# response
string image_location
